README.rst

```diff
@@ -43,7 +43,7 @@ being dropped.
 
 `Home page <http://matplotlib.org/>`_
 
-Installation
```

Comment:
Can you shorted the `===` lines as well?  I think sphinx is going to complain about the length mismatch.

Reply:
took so long 'cause apparently github will let me do the initial PR on my phone but won't let me edit the files on mobile. 🙁